core/base_operations.py

```
@@ -1,16 +1,19 @@
+from typing import Any, Dict, List, Optional
+
+
 class MathOperation:
     """Base class for all math operations."""
-    name = None
-    args = []
-    help = "Base operation"
+    name: Optional[str] = None
+    args: List[str] = []
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)